ModelFiles/dependencies.txt

```diff
@@ -1,6 +1,6 @@
-MATLAB	9.4.0.813654 (R2018a)
+MATLAB	9.1.0.441655 (R2016b)
 libSBML	5.17.0
-RAVEN_toolbox	2.0.2
+RAVEN_toolbox	unknown
```

Comment:
@feiranl the new release of RAVEN (2.0.2) is now working well with the model, could you try updating and see if only `dependencies.txt` changes? If so please commit that change to keep `devel` with RAVEN up to date.